langchain4j-azure-cosmos/src/main/java/dev/langchain4j/store/embedding/azure/cosmos/AzureCosmosDbMongoVCoreDocument.java

```diff
@@ -0,0 +1,23 @@
+package dev.langchain4j.store.embedding.azure.cosmos;
```

Comment:
it seems that `langchain4j-azure-cosmos-mongo-vcore` module naming suggests that there might be more db types under cosmos?
if yes, should package name follow the same convention? e.g. `dev.langchain4j.store.embedding.azure.cosmos.mongo.vcore`;

Reply:
updated the directory name